### PR TITLE
Add radio applet

### PR DESCRIPTION
**Pull request by Driglu4it**

### CODE REVIEW

**Comment by jaszhix** [comment]
There's some issues with the [file structure](https://github.com/linuxmint/cinnamon-spices-applets#file-structure) - ```info.json``` should look like:

```
{
    "author": "Driglu4it"
}
```

The applet should reside in a folder named ```radio@driglu4it```. In files/, you need to place the applet files in another folder named ```radio@driglu4it```.

File: radio/files/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "description": "This simple radio for Cinnamon. Require moc player. In setting you may add, edit and remove stations.",
+  "max-instances": 1,
+  "dangerous": false,
```
Comment (jaszhix) [inline radio/files/metadata.json]: The dangerous flag is automatically set by Cinnamon so this should be removed.

**Comment by Driglu4it** [comment]
How to rename folder, i can't insert symbol "@" in path (it replace for "-")?

**Comment by jaszhix** [comment]
You should be able to...

```mv radio radio@driglu4it```

**Comment by Driglu4it** [comment]
All repair )

**Pull request merged**